scripts/sct_label_utils.py

```diff
@@ -598,6 +602,30 @@ def launch_sagittal_viewer(self, labels):
 
         return output
 
+    def remove_or_keep_labels(self, labels, action):
+        """
+        Create or remove labels from self.image_input
+        :param list(int): Labels to add or remove
+        :param str: 'remove': remove labels (i.e. set to zero), 'keep': keep labels
+        """
+        image_output = msct_image.zeros_like(self.image_input) if action == 'keep' else  self.image_input.copy()
```

Comment:
for clarity, always use:
```python
if action == 'keep':
  image_output = msct_image.zeros_like(self.image_input)
elif action == 'remove':
  image_output = self.image_input.copy()
```

Reply:
Understood, changed in commit [70d3f2e](https://github.com/neuropoly/spinalcordtoolbox/commit/70d3f2e11c5153c5cd3a576a930e66dd48291ca7)